### ISSUE TITLE
"odo component describe component nodes" expected invalid "name: nodejs-web-app"

### ISSUE THREAD

**Issue by prietyc123** [openshift/odo#3465]
/kind bug

## What versions of software are you using?

**Operating System:**
All supported

**Output of `odo version`:**
master

## How did you run odo exactly?

Running `odo component describe component nodes` on travis CI.

## Actual behavior

```
Running odo with args [odo registry list]
[odo] NAME                       URL
[odo] DefaultDevfileRegistry     https://raw.githubusercontent.com/odo-devfiles/registry/master
[odo] RegistryName               https://raw.githubusercontent.com/odo-devfiles/registry/master
[odo] I0701 07:19:33.891821   18766 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo] I0701 07:19:33.891933   18766 odo.go:72] Could not get the latest release information in time. Never mind, exiting gracefully :)
Running odo with args [odo catalog describe component nodejs]
[odo] I0701 07:19:34.021100   18775 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo] I0701 07:19:34.022643   18775 cert_rotation.go:137] Starting client certificate rotation controller
[odo] I0701 07:19:34.025659   18775 context.go:301] There is NO config file that exists, we are however ignoring this as the ignoreMissingConfiguration flag has been passed in as true
[odo] I0701 07:19:34.038411   18775 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo] I0701 07:19:34.038613   18775 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo] I0701 07:19:34.039349   18775 catalog.go:324] Unable to get Image Streams from namespace openshift. Error unable to list imagestreams: the server could not find the requested resource (get imagestreams.image.openshift.io)
[odo] I0701 07:19:34.039943   18775 catalog.go:333] Unable to get Image Streams from namespace smcraugcdd. Error unable to list imagestreams: the server could not find the requested resource (get imagestreams.image.openshift.io)
[odo] I0701 07:19:34.039978   18775 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo] I0701 07:19:34.040075   18775 component.go:65] Please log in to an OpenShift cluster to list OpenShift/s2i components
[odo] I0701 07:19:34.183082   18775 preference.go:182] The path for preference file is /tmp/263188910/config.yaml
[odo]  ⚠  There are multiple components named "nodejs" in different multiple devfile registries.
[odo] 
[odo] Devfile Component(s):
[odo] 
[odo] * Registry: DefaultDevfileRegistry
[odo] I0701 07:19:34.239587   18775 content.go:32] converted devfile YAML to JSON
[odo] I0701 07:19:34.239724   18775 apiVersion.go:50] devfile apiVersion: '2.0.0'
[odo] I0701 07:19:34.239737   18775 context.go:53] devfile apiVersion '2.0.0' is supported in odo
[odo] I0701 07:19:34.243893   18775 schema.go:47] validated devfile schema
[odo] 
[odo] Starter Projects:
[odo] I0701 07:19:34.244263   18775 validate.go:35] Successfully validated devfile sections
[odo] ---
[odo] clonepath: ""
[odo] git:
[odo]   branch: ""
[odo]   location: https://github.com/odo-devfiles/nodejs-ex.git
[odo]   sparsecheckoutdir: ""
[odo]   startpoint: ""
[odo] github: null
[odo] name: nodejs-starter
[odo] zip: null
[odo] 
[odo] * Registry: RegistryName
[odo] I0701 07:19:34.299890   18775 content.go:32] converted devfile YAML to JSON
[odo] I0701 07:19:34.299981   18775 apiVersion.go:50] devfile apiVersion: '2.0.0'
[odo] I0701 07:19:34.299995   18775 context.go:53] devfile apiVersion '2.0.0' is supported in odo
[odo] I0701 07:19:34.303823   18775 schema.go:47] validated devfile schema
[odo] 
[odo] Starter Projects:
[odo] I0701 07:19:34.303927   18775 validate.go:35] Successfully validated devfile sections
[odo] ---
[odo] clonepath: ""
[odo] git:
[odo]   branch: ""
[odo]   location: https://github.com/odo-devfiles/nodejs-ex.git
[odo]   sparsecheckoutdir: ""
[odo]   startpoint: ""
[odo] github: null
[odo] name: nodejs-starter
[odo] zip: null
[odo] 
[odo] I0701 07:19:34.304025   18775 odo.go:72] Could not get the latest release information in time. Never mind, exiting gracefully :)
Deleting project: smcraugcdd
Running kubectl with args [kubectl delete namespaces smcraugcdd]
[kubectl] namespace "smcraugcdd" deleted
Setting current dir to: /home/travis/gopath/src/github.com/openshift/odo/tests/integration/devfile
Deleting dir: /tmp/263188910
• Failure [6.500 seconds]
odo devfile catalog command tests
/home/travis/gopath/src/github.com/openshift/odo/tests/integration/devfile/cmd_devfile_catalog_test.go:13
  When executing catalog describe component with a component name with multiple components
  /home/travis/gopath/src/github.com/openshift/odo/tests/integration/devfile/cmd_devfile_catalog_test.go:103
    should print multiple devfiles from different registries [It]
    /home/travis/gopath/src/github.com/openshift/odo/tests/integration/devfile/cmd_devfile_catalog_test.go:104
    Expected
        <string>: Devfile Component(s):
        
        * Registry: DefaultDevfileRegistry
        
        Starter Projects:
        ---
        clonepath: ""
        git:
          branch: ""
          location: https://github.com/odo-devfiles/nodejs-ex.git
          sparsecheckoutdir: ""
          startpoint: ""
        github: null
        name: nodejs-starter
        zip: null
        
        * Registry: RegistryName
        
        Starter Projects:
        ---
        clonepath: ""
        git:
          branch: ""
          location: https://github.com/odo-devfiles/nodejs-ex.git
          sparsecheckoutdir: ""
          startpoint: ""
        github: null
        name: nodejs-starter
        zip: null
        
        
    to contain substring
        <string>: name: nodejs-web-app
```

## Expected behavior

The [output](https://github.com/openshift/odo/blob/master/tests/integration/devfile/cmd_devfile_catalog_test.go#L109) of `odo component describe component nodes` should contain "name: nodejs-starter"

## Any logs, error output, etc?

Logs : https://travis-ci.com/github/openshift/odo/jobs/356174343#L447

**Comment by prietyc123** [comment]
It has been fixed in #3459

**Comment by kadel** [comment]
/close
addressed in  #3459
